integration_tests/test_svn_pull_update.py

```diff
@@ -0,0 +1,77 @@
+# -*- Mode:Python; indent-tabs-mode:nil; tab-width:4 -*-
+#
+# Copyright (C) 2016 Canonical Ltd
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License version 3 as
+# published by the Free Software Foundation.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+import os
+import subprocess
+import shutil
+
+from testtools.matchers import FileExists
+
+import integration_tests
+
+
+class SubversionSourceTestCase(integration_tests.TestCase):
+
+    def setUp(self):
+        super().setUp()
+        if shutil.which('svn') is None:
+            self.skipTest('svn is not installed')
+
+    def _init_svn(self):
+        subprocess.check_call(
+            ['svnadmin', 'create', 'repo'], stdout=subprocess.DEVNULL)
+
+    def test_pull_svn(self):
```

Comment:
You can put this test in the same file as the other. Call one test_pull_svn_checkout and the other test_pull_svn_update.